recipes/jasper/all/conanfile.py

```diff
@@ -13,10 +13,16 @@ class JasperConan(ConanFile):
     exports_sources = ["CMakeLists.txt", "patches/**"]
     generators = "cmake", "cmake_find_package"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False],
-               "fPIC": [True, False],
-               "jpegturbo": [True, False]}
-    default_options = {"shared": False, "fPIC": True, "jpegturbo": False}
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_libjpeg": ["libjpeg", "libjpeg-turbo"],
```

Comment:
I don't like the change options after having the recipe available, but it seems correct. The previous options jpegturbo=False sounds like we don't offer any jpeg support, thus this update looks right.

Reply:
Yeah, it will only impact people explicitly setting the `jpegturbo` option though.